src/main/java/seedu/club/model/ClubBook.java

```diff
@@ -96,6 +99,11 @@ public void resetData(ReadOnlyClubBook newData) {
         List<Member> syncedMemberList = newData.getMemberList().stream()
                 .map(this::syncWithMasterTagList)
                 .collect(Collectors.toList());
+        if (newData.getLoggedInMember() != null) {
```

Comment:
Is the if else necessary? Is it possible to just use setLoggedInMember(newData.getLoggedInMember());